lib/cli.rb

```diff
@@ -127,6 +136,14 @@ def define_options(parser)
         reject_report_sections_option(parser)
         boot_device_option(parser)
         allow_test_duplication_option(parser)
+        share_on_host_net_option(parser)
+      end
+
+      def share_on_host_net_option(parser)
+        parser.on('--share-on-host-net <share_on_host_net>', String,
```

Comment:
It is a misnomer. We don't share a directory on the host network. We share a directory.